src/tasks/publish/getDryRunOutput.ts

```diff
@@ -0,0 +1,93 @@
+import {
+  PublishVersionTxData,
+  encodePublishVersionTxData
+} from '~/src/utils/apm'
+import { getAppNameParts } from '~/src/utils/appName'
+
+interface DataItem {
+  name: string
+  value: string
+}
+
+/**
+ * Returns a string with formated data about a publish version tx
+ */
+export function getDryRunOutput({
+  txData,
+  appName,
+  nextVersion,
+  bump,
+  contractAddress,
+  contentHash,
+  ipfsGateway,
+  rootAccount
+}: {
+  txData: PublishVersionTxData
+  appName: string
+  nextVersion: string
+  bump: string
+  contractAddress: string
+  contentHash: string
+  ipfsGateway: string
+  rootAccount: string
+}): string {
+  const { registryName } = getAppNameParts(appName)
+  let action: string
+  const params: DataItem[] = []
+  const addData = (name: string, value: string): void => {
+    params.push({ name, value })
+  }
+
+  switch (txData.methodName) {
```

Comment:
The `dry-run` is a good place to use some color for the output, what you think?

Reply:
I've added formatting and color for dry runs and executions with 4ee9f0a